torch_models/pytest_iree/git_lfs.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import logging
+import subprocess
+from pytest_iree.artifact import Artifact
+
+logger = logging.getLogger(__name__)
+
+class GitLfsArtifact(Artifact):
+    """
+    Represents an artifact that is stored in the repo using git lfs.
+    """
```

Comment:
Can you add details on the directory structure it creates like other artifacts have?